public/scss/_main.scss

```diff
@@ -0,0 +1,143 @@
+/* ----------------------------------------------- */
+/* BASIC SETUP */
+/* ----------------------------------------------- */
+
+$primary-color: #d10157;
```

Comment:
move all these variables to a separate file. Also, I can see below that you have used hardcoded values , add those also to the variables

Reply:
Done.